s3/pkg/service/object.go

```diff
@@ -296,8 +262,9 @@ func (s *s3Service) GetObject(ctx context.Context, req *pb.GetObjectInput, strea
 		log.Errorln("failed to get object info from meta storage. err:", err)
 		return err
 	}
-	_, _, err = CheckRights(ctx, object.TenantId)
```

Comment:
Does the ACL matters here?

Reply:
@sfzeng , I checked the code again, and found the CheckRights here is not necessary, and I will delete it

Reply:
Hi @sunfch What do you mean by 'not necessary'? Isn't it necessary to check ACL when get object?

Reply:
@sfzeng  I found that CheckRights is called twice here, so I delete one